src/libertem/io/dataset/blo.py

```diff
@@ -164,17 +164,22 @@ def initialize(self, executor):
         return self
 
     @classmethod
-    def detect_params(cls, path, executor):
+    def detect_params(cls, path, executor, info=False):
         try:
             ds = cls(path, endianess='<')
             ds = ds.initialize(executor)
             if not executor.run_function(ds.check_valid):
                 return False
-            return {
+            parameters = {
                 "path": path,
                 "tileshape": (1, 8) + tuple(ds.shape.sig),
                 "endianess": "<",
             }
+            additional_info = {}
+            # If additional info is required
+            if info:
+                parameters.update(additional_info)
+            return parameters
```

Comment:
Hmm, to make the datasets easier, what do you think about always returning the info? And then decide in `detect` if we want to return it or not. Then we have a single place where the decision is made, and not one in each dataset.

Reply:
Then what do you suggest I do when the dataset's `detect_params` is called directly? [Example](https://github.com/LiberTEM/LiberTEM/blob/master/tests/io/test_hdf5.py#L247)

Reply:
Those places need to be updated - other than tests, `detect_params` should only be called in the `detect` function AFAIK

Reply:
Done!